infra/recipes/docker-compose/common/debug-profile/housetables.yml

```diff
@@ -3,4 +3,4 @@ services:
     ports:
       - 6005:6005
     environment:
-      - JAVA_TOOL_OPTIONS=-agentlib:jdwp=transport=dt_socket,server=y,address=6005,suspend=n
\ No newline at end of file
+      - JAVA_TOOL_OPTIONS=-agentlib:jdwp=transport=dt_socket,server=y,address=*:6005,suspend=n
```

Comment:
these are required to enable debugging on my machine, i'm not sure why no one else needs them, but i'm going ahead and adding them for everyone's assistance.